src/atoms/feedCardItem/icons.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import {
```

Comment:
@shan8851  there is a situation , with a dark theme , ask Michal for should we do
![Screen Shot 2022-06-19 at 10 29 46](https://user-images.githubusercontent.com/35912161/174470645-2ec9cc8d-34e5-4c60-ade2-7b2ca61067d1.png)

Reply:
@goshatravin I have spoken with Michal about this, she said there are too many tokens to make our own lib of them, she is thinking maybe we can do something on hover, although this would not fix the default view.